ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/Mode.scala

```diff
@@ -7,9 +7,15 @@ sealed abstract class Mode
 
 object Mode {
 
-  /** Run the participant */
+  /** Run the participant, accepts HOCON configuration */
   case object Run extends Mode
 
+  /** Run the participant in legacy mode with accepted CLI arguments */
+  case object RunLegacy extends Mode
```

Comment:
Can you make it more explicit that `RunLegacy` refers to the config feed mode? Maybe naming it `RunLegacyCliConfig` or something like that

Reply:
would you also consider renaming command to run in this mode?

Reply:
You mean changing from `RunLegacy` to just `Run` or..? I don't fully understand your question

Reply:
So with CLI currently in order to trigger `Run` mode you do the following:
`./sox run -c myconfig.conf`
for `RunLegacy`:
`./sox run-legacy --participant=participant-id=example,port=0....`

so would you change `run-legacy` command to something else? `run-legacy-cli-config` ?